rudder-web/src/main/webapp/style/rudder/rudder-menu.css

```diff
@@ -118,6 +118,9 @@ pre.json-beautify, .tw-bs pre.json-beautify{
   position: relative;
   margin-bottom: 0;
   padding-right: 25px;
+  white-space: pre-wrap;
```

Comment:
What is the impact of those addition on Node properties display ?

Reply:
There was nearly the same bug on the Node properties, we couldn't see the end of a long string (see attached file). This css rule fix this bug too !
![nodeproperties](https://user-images.githubusercontent.com/9928447/31334632-5706cc46-acef-11e7-8916-06caab5d5f8b.png)

Reply:
Great !!